doc/guides/modules.rst

```diff
@@ -153,8 +175,32 @@ Build settings supported in the :file:`module.yml` file are:
   build system. Additional architectures must be located in a
   :file:`<arch_root>/arch` folder.
 
-Example of a :file:`module.yaml` file containing additional roots, and the
-corresponding file system layout.
+Example of a :file:`module.yaml` file containing additional roots
+corresponding to the directory structure of the Zephyr repository
+put beneath the :file:`zephyr` subdirectory.
+
+.. code-block:: yaml
+
+   build:
```

Comment:
Perhaps we could use another folder instead of `zephyr`, to be more explicit?

Reply:
> Perhaps we could use another folder instead of `zephyr`, to be more explicit?

I want to bikeshed the chosen alternative 'zephyrous' so hard, but I won't :smile: